app/backend3/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM mcr.microsoft.com/dotnet/sdk:7.0 AS build-env
```

Comment:
so the dotnet stays and is moved to backed3?

Reply:
exactly, but not used, it's just there if someone wants to expand on it later